libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/FormatUtils.java

```diff
@@ -32,4 +32,27 @@ public static final String formatInt(int value) {
     public static final String formatDecimal(int value) {
         return DECIMAL_INSTANCE.get().format(value).toString();
     }
+
+    /*
+     * returns the passed long formatted has an human readable filesize. Ex: 10 GB
+     */
+    public static final String formatFileSize(long size) {
+        if (size <= 0) {
+            return "0";
+        }
+        final String[] units = new String[] {"B", "kB", "MB", "GB", "TB" };
+        int digitGroups = (int) (Math.log10(size) / Math.log10(1024));
+
+        return new DecimalFormat("#,##0.#").format(size / Math.pow(1024, digitGroups)) + " " + units[digitGroups];
+    }
+
+    /*
+     * returns the passed double percentage (0 to 1) formatted has an human readable percentage. Ex: 0.25 returns 25%
+     */
+    public static final String formatPercentage(double value) {
+        NumberFormat percentFormat = NumberFormat.getPercentInstance();
+        percentFormat.setMaximumFractionDigits(1);
+        String result = percentFormat.format(value);
+        return result;
```

Comment:
we might here just get rid of the `String result` declaration and combine these two lines into `return percentFormat.format(value);`

Reply:
addressed in d2fc531